### PR TITLE
Correct the URLs in package.json

### PR DESCRIPTION
**Pull request by peternewman**
Given this is a separate module

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,15 +3,15 @@
 	"version": "1.0.0",
 	"private": true,
```
Comment (peternewman) [inline package.json:4]: Is that intentional given
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private
and
https://www.npmjs.com/package/samsung-tv-remote

**Review by Badisi — APPROVED** [review]

**Comment by peternewman** [comment]
Thanks @Badisi .

Can you publish a new release to fix the links on npm too please:
https://www.npmjs.com/package/samsung-tv-remote

**Comment by Badisi** [comment]
Hello @peternewman,

Sorry for the late reply but this library was quite old and needed a little refresh ^^

[Version 2.0.0](https://github.com/Badisi/samsung-tv-remote/releases/tag/2.0.0) was just released 🚀🚀 - so you can go check it out and tell me what you think !

And thanks a lot for the contribution and support 👍

**Comment by peternewman** [comment]
Thanks @Badisi . I've actually got an older (pre 2016 Samsung TV), so can't really test it. My interest was really just around removing the confusion for people trying to use other code to control the older TVs!

**Pull request merged**